TriblerGUI/widgets/downloadspage.py

```diff
@@ -431,13 +473,16 @@ def on_right_click_item(self, pos):
             self.selected_items.append(item_clicked)
 
         menu = TriblerActionMenu(self)
+        menu.setMinimumWidth(CONTEXT_MENU_WIDTH)
```

Comment:
Cool, I didn't know about this method 👍 . Should we add this to the `TriblerActionMenu` object itself maybe?

Reply:
Moved to TriblerActionMenu